### PR TITLE
Fix/git ignore

### PR DESCRIPTION
**Pull request by ankitmashu**

### CODE REVIEW

**Comment by karun-singh** [comment]
retest this please

**Comment by ankitmashu** [comment]
retest this please

**Comment by ananjaykumar2** [comment]
retest this please

**Comment by ananjaykumar2** [comment]
retest this please

**Comment by ankitmashu** [comment]
retest this please

**Review by pranavrd — APPROVED** [review]
LGTM

File: SETUP.md [hunk 1]
```diff
@@ -28,12 +28,31 @@ The Resource Server also connects with various DX dependencies namely
 - Authorization Server : used to download the certificate for token decoding
 - Catalogue Server : used to download the list of resources, access policies and query types supported on a resource.
 - Auditing Server : used to store metering related data.
+
+## Create configs files
+
+Make a copy of sample configs directory and add appropriate values to all files.
+
+```console
+ cp -r example-configs/* .
```
[the remark this answers is not in the document]
Comment (ankitmashu) [reply SETUP.md:37]: solved in pr #516

**Pull request merged**